sources/us/fl/miami2.json

```diff
@@ -0,0 +1,34 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "12086",
+            "name": "Miami-Dade County",
+            "state": "Florida"
+        },
+        "country": "us",
+        "state": "fl",
+        "county": "Miami-Dade"
+    },
+    "attribution": "Miami-Dade County",
+    "data": "https://gisweb.miamidade.gov/arcgis/rest/services/MD_Emaps/MapServer/72",
```

Comment:
This source has addresses that the other source does not have. I was debating what to do here, as this source actually have slightly fewer addresses in total -- so I decided to keep them both. Let me know what you guys think.

Specifically, I was looking at `16055 SW 117th Ave` which is present in this source but not in the source from `us/fl/miami.json`